.idea/runConfigurations/MCreator.xml

```diff
@@ -1,7 +1,7 @@
 <component name="ProjectRunConfigurationManager">
```

Comment:
This change does not belong to this PR.

Also my IDEA 2022.2 installation shows this with your change:

![image](https://user-images.githubusercontent.com/16374228/183583317-f22e504c-ab63-4ab4-8e6b-be364ddac773.png)

![image](https://user-images.githubusercontent.com/16374228/183583348-fd84a3e1-f006-4e59-802e-4ebb8ddada58.png)

![image](https://user-images.githubusercontent.com/16374228/183583386-d2eacae8-0404-425c-a976-5e4bd9738a3b.png)

As no other contributor is having this issue, we can't accept this change.

Module names also never really included package name to my knowledge.

Reply:
> Also my IDEA 2022.2 installation shows this with your change:
![image](https://user-images.githubusercontent.com/38427877/183660592-a7d7f7f5-b9a2-40fe-8662-f0a24d361e94.png)

My editor shows this error when I don't have this change... :/

Reply:
I tried to re-install Intellij and then, re-clone my fork and weirdly, Intellij fixed it... once again, it was a problem of Intellij...😔

Reply:
Hmm, let's hope they fix this asap, until then you may have to not add those files every commit :/

Reply:
> Hmm, let's hope they fix this asap, until then you may have to not add those files every commit :/

I don't need it anymore. The problem is resolved.